### PR TITLE
Cleanup stylelint comments

### PR DESCRIPTION
**Pull request by MartijnCuppens**
`stylelint-disable` and `stylelint-enable` is used a lot while this can also be achived with `stylelint-disable-next-line` 

I've also added some comments to clarify why I made some changes.

### CODE REVIEW

File: scss/_close.scss [hunk 1]
```diff
@@ -25,11 +25,10 @@
 // If you want the anchor version, it requires `href="#"`.
 // See https://developer.mozilla.org/en-US/docs/Web/Events/click#Safari_Mobile
 
-// stylelint-disable property-no-vendor-prefix, selector-no-qualifying-type
+// stylelint-disable-next-line selector-no-qualifying-type
 button.close {
   padding: 0;
   background-color: transparent;
   border: 0;
-  -webkit-appearance: none;
```
Comment (MartijnCuppens) [inline scss/_close.scss]: `autoprefixer` takes care of this

File: scss/_reboot.scss [hunk 2]
```diff
@@ -163,12 +161,10 @@ dfn {
   font-style: italic; // Add the correct font style in Android 4.3-
 }
 
-// stylelint-disable font-weight-notation
```
Comment (MartijnCuppens) [inline scss/_reboot.scss]: There was no need for this (see https://github.com/stylelint/stylelint/tree/master/lib/rules/font-weight-notation#font-weight-notation)

File: scss/_dropdown.scss [hunk 3]
```diff
@@ -105,7 +105,7 @@
 }
 
 // When enabled Popper.js, reset basic dropdown position
-// stylelint-disable no-duplicate-selectors
+// stylelint-disable-next-line no-duplicate-selectors
```
Comment (XhmikosR) [inline scss/_dropdown.scss]: I was hoping we could reduce the amount of duplicate selectors when possible later :)

**Review by XhmikosR — APPROVED** [review]
LGTM, thanks!

**Pull request merged**